### ISSUE TITLE
Missing Favicon

### ISSUE THREAD

**Issue by so-rose** [dtudk/pythonsupport-page#18]
The site could use a 'favicon.ico' for pretty titlebar icons.

**Comment by zerothi** [comment]
Yes, but we don't have any as of yet. DTU regulation are that we cannot mix the DTU logo with anything. :(

**Comment by so-rose** [comment]
What about this one?

<https://publicdomainvectors.org/en/free-clipart/Computer-host-icon-vector-image/13191.html>

It's CC0 and all.
